src/drawentity.cpp

```diff
@@ -192,7 +212,11 @@ bool Entity::CanBeDragged() const {
         && (timesApplied == 0)) return false;
     // for these types of entities the first point will indicate draggability
     if(HasEndpoints() || type == Entity::Type::CIRCLE) {
-        return SK.GetEntity(point[0])->CanBeDragged();
```

Comment:
Hmm if removed from here `CanBeDragged` remains called in only one place in the code.... I've never looked in these parts much... I need to think....